### DISCUSSION TITLE
Schematic for the IMSAI 80080 board?

### DISCUSSION THREAD

**Discussion by GaryBoone** [thehighnibble/thehighnibble.github.io#101] [IMSAI8080esp replica]
I'm debugging SMD soldering, so would like to check the schematic for the IMSAI 80080 board. Is there one available?

**Comment by GrantMeStrength** [comment]
I don't think Mr The High Nibble has published that, as is his prerogative.

**Comment by bertelschmitt** [comment]
There really should be a schemactic.  I spent a whole day tracing out how the  SD interface is hooked up to the esp32, because I am one of the many who botched the hand soldering of a surface-mount part. After a day of tracing, I found out that I could insert a ready-made SD-I/F  with a pin header into the through-holes below RN1. It really shouldn't be that hard. There is not a lot of intellectual property protected by not publishing the schematic. A few days wasted with a multimeter, and the job would be done.

**Comment by bertelschmitt** [comment]
I have received an email from Dave, saying that schematics will be made available  on request.
